pkgs/development/ocaml-modules/callipyge/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+
+, alcotest
+, eqaf
+, fmt
+}:
+
+buildDunePackage rec {
+  pname = "callipyge";
+  version = "0.2";
+
+  src = fetchFromGitHub {
```

Comment:
Use <https://github.com/oklm-wsh/Callipyge/releases/download/v0.2/callipyge-0.2.tbz> here which is the proper release tarball also used by OPAM.